test.dockerfile

```diff
@@ -35,7 +35,8 @@ RUN pip install tox
 # Install the relevant dependencies. Keep printing so Travis knows we're alive.
 RUN ["bash", "-c", "( while true; do echo '.'; sleep 60; done ) & tox --notest"]
 
-# Finally, upload our actual code!
+# Finally, cleans cached code and upload our actual code!
+RUN rm -rf ./gym*
```

Comment:
Yeah, I thought that might happen. However, I think `rm -rf ./` won't catch dotfiles. You probably want to `cd` out of the currently directory.